### ISSUE TITLE
Ubuntu 18.04 - "Error: No such container:"

### ISSUE THREAD

**Issue by Tw33ki** [shellhub-io/shellhub#3445]
### Description

Hi there, 
i tried to use Ubuntu 18.04 as Client for Shellhub. I've installed it like ever, but for some reason i get this Message: 
```
root@server:/var/lib/docker# curl -sSf "https://shellhub.XXXX.de/install.sh?tenant_id=44774cb6-6f5f-44fb-a017-XXXXXX" | sh
Unable to find image 'shellhubio/agent:v0.13.6' locally
v0.13.6: Pulling from shellhubio/agent
e0b29c293021: Pull complete
c53ce6574aac: Pull complete
011092fecc77: Pull complete
633e2ee5ed8e: Pull complete
df1008174658: Pull complete
00d852a2a9c6: Pull complete
35eb6e7313fe: Pull complete
Digest: sha256:f8318f2db8f16ab7225677a795a2f69f3c69afc646ebc07d817a362481a42861
Status: Downloaded newer image for shellhubio/agent:v0.13.6
a3c414173a888fb092325182c81b147670056c4111af4907c0d0a1af5750bf56
root@server:/var/lib/docker# docker ps
CONTAINER ID   IMAGE     COMMAND   CREATED   STATUS    PORTS     NAMES
root@server:/var/lib/docker# docker ps -a
CONTAINER ID   IMAGE                      COMMAND        CREATED         STATUS                     PORTS     NAMES
a3c414173a88   shellhubio/agent:v0.13.6   "/bin/agent"   4 seconds ago   Exited (0) 3 seconds ago             shellhub
root@server:/var/lib/docker# docker logs a3c414173a88
time="2024-01-15T13:11:43Z" level=info msg="Setting log level" log_level=info
time="2024-01-15T13:11:43Z" level=warning msg="Error: No such container: "
```

Informations to my Client: 
```
`root@server:/var/lib/docker# docker version
Client: Docker Engine - Community
 Version:           24.0.2
 API version:       1.43
 Go version:        go1.20.4
 Git commit:        cb74dfc
 Built:             Thu May 25 21:52:13 2023
 OS/Arch:           linux/amd64
 Context:           default

Server: Docker Engine - Community
 Engine:
  Version:          24.0.2
  API version:      1.43 (minimum version 1.12)
  Go version:       go1.20.4
  Git commit:       659604f
  Built:            Thu May 25 21:52:13 2023
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.6.21
  GitCommit:        3dce8eb055cbb6872793272b4f20ed16117344f8
 runc:
  Version:          1.1.7
  GitCommit:        v1.1.7-0-g860f061
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
root@server:/var/lib/docker# lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 18.04.6 LTS
Release:        18.04
Codename:       bionic
`
```

Will be there support or workaround for this problem?
Thanks in advance

### Edition

Community

### Version

v0.13.6

**Comment by heiytor** [comment]
Could you please test it with the tag [v0.14.0-rc.6](https://github.com/shellhub-io/shellhub/tree/v0.14.0-rc.6)?

**Comment by heiytor** [comment]
I'm glad to hear that! We have just released the official [0.14.0](https://github.com/shellhub-io/shellhub/releases/tag/v0.14.0), and you can use it if you prefer.
